apps/users/routers.py

```diff
@@ -226,30 +231,31 @@ async def create_student(
     auth=Depends(get_current_user),
     session: AsyncSession = Depends(get_async_session),
 ):
-    """Method for create university student record.
+    """**Method for university student record creation.**
 
-    Path:
-        - university_id: university id
+    **Path**:
+    - **university_id**: university id
 
-    Input:
-        - last_name: student last name, required
-        - first_name: student fist name, required
-        - middle_name: student middle name
-        - telephone_number: student telephone number, must consists of 12 digits,
-            required
-        - course_id: student course id, must be between 1 and 6, required
-        - faculty_id: faculty id, required
-        - speciality_id: speciality id, required
-        - gender: student gender, 'Ч' or 'Ж', required
+    **Input**:
+    - **last_name**: student last name, required
+    - **first_name**: student fist name, required
+    - **middle_name**: student middle name
+    - **telephone_number**: student telephone number, must consists of 12 digits,
+        required
+    - **course_id**: student course id, must be between 1 and 6, required
+    - **faculty_id**: faculty id, required
+    - **speciality_id**: speciality id, required
+    - **gender**: student gender, 'Ч' or 'Ж', required
 
-    Return: created student id
+    **Return**:
+    - created student id
     """
     student = await user_handler.create_student(
         request=request, student=student, session=session
     )
     return {
         "data": student,
-        "message": f"Created student with id {student.get('student_id')}",
+        "message": f"Created student with id {student.student_id}",
```

Comment:
are u sure that it is works? bcs as i can remember u fixed it in ticket EU-122
please double check

Reply:
Everything is working fine.